src/WorldHandler.cpp

```diff
@@ -15,21 +14,21 @@ void WorldHandler::start() {
     proto::SSL_Referee ref_packet;
 
     roboteam_utils::Timer t;
+
     t.loop([&]() {
       handleVisionPackets(vision_packet);
       handleRefboxPackets(ref_packet);
 
-      KF->kalmanUpdate();
-      world_pub->send(KF->getWorld());
+      world_pub->send(KF->getWorld(lastPacketTime));
     }, 100);
 }
 
 void WorldHandler::init() {
-    world = new WorldBase();
-    KF = new KalmanFilter;
+    KF = new WorldFilter;
```

Comment:
I propose a new name for KF: kalmanFilter

Reply:
I renamed it to worldFilter which is much more clear.